app/src/controllers/client/review/ReviewController.ts

```diff
@@ -9,6 +9,8 @@ export type ReviewServiceInterface = {
   fetchReviewsWithTotalCountAndShopNameAndClientName(user: UserForAuth | undefined, shopId: number, page?: number,
     order?: OrderBy, take?: number): Promise<{ reviews:
       (Review & { shopName: string, clientName: string })[], totalCount: number }>
+  deleteReview(user: UserForAuth | undefined, shopId: number, reviewId: number)
```

Comment:
since this is a protected route, the use should already be checked here, maybe use a controller error for authorization error?